### PR TITLE
Update: `Browse building floors` the way to select first floor

### PR DESCRIPTION
**Pull request by yo1995**
## Description

This PR implements `Browse building floors` in `maps` category, to address several feedback from Nathan. README will be updated when the design PR gets merged

## Linked Issue(s)

`common-samples/issues/3321`

## How To Test

The sample should behave exactly the same as before.

### CODE REVIEW

**Review by vquach2404 — APPROVED** [review]
Ready for the next reviewer!

**Review by vquach2404 — APPROVED** [review]

**Review by nCastle1 — APPROVED** [review]
Looks great!

File: arcgis-ios-sdk-samples/Maps/Browse building floors/README.md [hunk 1]
```diff
@@ -14,14 +14,11 @@ Use the picker to browse different floor levels in the facility. Only the select
...
-1. Create an `AGSPortalItem` using the identifier of a floor-aware web map.
-2. Create a map using the portal item.
-3. Create a map view and assign the map to it.
-4. Wait for the map to load and retrieve the map's `floorManager` property.
-5. Wait for the floor manager to load and retrieve the floor-aware data.
-6. Set all floors to not visible.
-7. Set only the selected `AGSFloorLevel` to visible using the `isVisible` property of the floor level.
-* **Note:** Manually set the default floor level to the first floor.
+1. Create and load a floor-aware web map using the identifier of an `AGSPortalItem`.
+2. Load the map and retrieve the map's `floorManager` property. Check that the map has a `floorManager` or `floorDefinition` property to ensure the map is floor-aware.
```
Comment (mikewilburn) [inline arcgis-ios-sdk-samples/Maps/Browse building floors/README.md:18]: 👍

File: arcgis-ios-sdk-samples/Maps/Browse building floors/BrowseBuildingFloorsViewController.swift [hunk 2]
```diff
@@ -95,7 +95,11 @@ class BrowseBuildingFloorsViewController: UIViewController {
     /// Called after the floor manager of the web map is loaded without error.
     func floodManagerDidLoad(_ floorManager: AGSFloorManager) {
         guard let geometry = floorManager.sites.first?.geometry,
-              let firstFloor = floorManager.levels.first(where: { $0.longName == "Level 1" }) else { return }
+              // Select the ground floor using `verticalOrder`.
+              // In the case of buildings with basements, they can have negative
+              // vertical orders; ground floor is expected to be 0.
+              // You can also use level ID, number or name to locate a floor.
+              let firstFloor = floorManager.levels.first(where: { $0.verticalOrder == 0 }) else { return }
```
Comment (mikewilburn) [inline arcgis-ios-sdk-samples/Maps/Browse building floors/BrowseBuildingFloorsViewController.swift:102]: This seems like a very useful revision and I know it was pointed out in the common design. Extra comments are nice for clarity, too.

**Review by mikewilburn — APPROVED** [review]

**Pull request merged**